src/domUpdates.js

```diff
@@ -144,137 +159,23 @@ let domUpdates = {
-//       src='https://image.flaticon.com/icons/svg/32/32339.svg' alt='Add to
-//       recipes to cook'></button>
-//       <label for='favorite-button' class='hidden'>Click to favorite recipe
-//       </label>
-//       <button id='${recipe.id}' aria-label='favorite-button' class='favorite favorite-active card-button'>
-//       </button></header>
-//       <span id='${recipe.id}' class='recipe-name'>${recipe.name}</span>
-//       <img id='${recipe.id}' tabindex='0' class='card-picture'
-//       src='${recipe.image}' alt='Food from recipe'>
-//       </div>`)
-//     })
-//   }
-// }
-
-
-
-// function favoriteCard(event) {
-//   let specificRecipe = cookbook.recipes.find(recipe => {
-//     if (recipe.id  === Number(event.target.id)) {
-//       return recipe;
-//     }
-//   })
-//   if (!event.target.classList.contains('favorite-active')) {
-//     event.target.classList.add('favorite-active');
-//     favButton.innerHTML = 'View Favorites';
-//     user.addToFavorites(specificRecipe);
-//   } else if (event.target.classList.contains('favorite-active')) {
-//     event.target.classList.remove('favorite-active');
-//     user.removeFromFavorites(specificRecipe)
-//   }
-// }
-
-// function cardButtonConditionals(event) {
-//   if (event.target.classList.contains('favorite')) {
-//     favoriteCard(event);
-//   } else if (event.target.classList.contains('card-picture')) {
-//     displayDirections(event);
-//   } else if (event.target.classList.contains('home')) {
-//     favButton.innerHTML = 'View Favorites';
-//     populateCards(cookbook.recipes);
-//   }
-// }
-
-
-// function displayDirections(event) {
-//   let newRecipeInfo = cookbook.recipes.find(recipe => {
-//     if (recipe.id === Number(event.target.id)) {
-//       return recipe;
-//     }
-//   })
-//   let recipeObject = new Recipe(newRecipeInfo, ingredientsData);
-//   let cost = recipeObject.calculateCost()
-//   let costInDollars = (cost / 100).toFixed(2)
-//   cardArea.classList.add('all');
-//   cardArea.innerHTML = `<h3>${recipeObject.name}</h3>
-//   <p class='all-recipe-info'>
-//   <strong>It will cost: </strong><span class='cost recipe-info'>
-//   $${costInDollars}</span><br><br>
-//   <strong>You will need: </strong><span class='ingredients recipe-info'></span>
-//   <strong>Instructions: </strong><ol><span class='instructions recipe-info'>
-//   </span></ol>
-//   </p>`;
-//   let ingredientsSpan = document.querySelector('.ingredients');
-//   let instructionsSpan = document.querySelector('.instructions');
-//   recipeObject.ingredients.forEach(ingredient => {
-//     ingredientsSpan.insertAdjacentHTML('afterbegin', `<ul><li>
-//     ${ingredient.quantity.amount.toFixed(2)} ${ingredient.quantity.unit}
-//     ${ingredient.name}</li></ul>
-//     `)
-//   })
-//   recipeObject.instructions.forEach(instruction => {
-//     instructionsSpan.insertAdjacentHTML('beforebegin', `<li>
-//     ${instruction.instruction}</li>
-//     `)
-//   })
-// }
```

Comment:
Thanks for cleaning up this commented-out stuff. Next time make sure to mention this in your description of the PR though.